Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/DiagnosticInformationCollectioningRequestErrorDetectionStrategy.cs

```diff
@@ -18,7 +18,7 @@
 using Coypu;
 using Remotion.Web.Development.WebTesting.RequestErrorDetectionStrategies;
 
-namespace Remotion.Web.Development.WebTesting.IntegrationTests
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure
```

Comment:
The first one is alright, but I think I need some input for the second one. I don't understand where the error detection strategy comes from and (without the current implementation) constantly get errors that the strategy could not be loaded.

Reply:
error detection strategy:
Short Version:
we should be able to remove `_aspNetRequestErrorDetectionScope` in `IntegrationTest` of the `RequireUI` project since this is not set in ObjectBinding.Web either.

Long version: need to see the errors live.